### ISSUE TITLE
Only extract TFT relevant data

### ISSUE THREAD

**Issue by un0btanium** [stelar7/lol-parser#7]
Hello,

I am curious if it would be possible to only have `testCDragonWAD()` extract the data relevant to TFT since thats currently the only data I am interested in. I dont always want to generate the full 16GB+ of data on my harddrive (or does it only change the data that changed in the newest patch and I dont have to delete the old folder every time?).

**Comment by stelar7** [comment]
sure, i can set that up. fixing it when i get back from work in 9~ hours

**Comment by un0btanium** [comment]
Thats good to hear! No worries, this has time.

On another note: Currently the data is being saved into a folder called "pbe". Just to make sure, this does not actually extract pbe data from my lol client (I dont even have PBE Client installed :D). This does extract always the data from the live version when I update my LoL Client, correct?

edit: Because I tried to extract the newest data for patch 9.14.1b and it still shows the stats from 9.14.1a. But it also seems like the TFT patch was hotfixed in since I did not see a patch being downloaded. Do I need to play one round for the data to be updated? :S
